release/release_logs/2.2.0/stress_tests/stress_test_dead_actors.json

```diff
@@ -1,14 +1,14 @@
 {
-    "avg_iteration_time": 1.9442666888237,
-    "max_iteration_time": 9.263823747634888,
-    "min_iteration_time": 0.8118352890014648,
+    "avg_iteration_time": 2.066628179550171,
+    "max_iteration_time": 13.97415018081665,
```

Comment:
51% regression

Reply:
This is a `max` value, so high variance. I don't think we need to investigate since we don't know how widespread this is.